extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -13,3 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-slider-input-container {
+  background: var(--i-amphtml-interactive-option-background-color) !important;
+  border-radius: 2em !important;
+  box-shadow: 0.125em 0.25em 0.5em rgba(0, 0, 0, 0.1) !important;
```

Comment:
I think there's `box-shadow: var(--i-amphtml-interactive-component-shadow) !important;` that you can use so you don't specify new values